src/model/window-guardian.ts

```diff
@@ -66,18 +66,67 @@ const makeWindowGuardianConfig = (
     } as WindowGuardianConfig;
 };
 
+export const makeGuardianBrowserCAPI = (CAPI: CAPIType): CAPIBrowserType => ({
+    config: {
+        isDev: process.env.NODE_ENV !== 'production',
+        ajaxUrl: CAPI.config.ajaxUrl,
+        shortUrlId: CAPI.config.shortUrlId,
+        pageId: CAPI.config.pageId,
+        isPaidContent: !!CAPI.config.isPaidContent,
+        showRelatedContent: CAPI.config.showRelatedContent,
+        keywordIds: CAPI.config.keywordIds,
+        ampIframeUrl: CAPI.config.ampIframeUrl,
+        // commercialBundleUrl is only used in SSR
+        // commercialBundleUrl: CAPI.config.commercialBundleUrl,
+
+        // switches
+        cmpUi: CAPI.config.switches.cmpUi,
+        slotBodyEnd: CAPI.config.switches.slotBodyEnd,
+        ampPrebid: CAPI.config.switches.ampPrebid,
+        permutive: CAPI.config.switches.permutive,
+
+        // used by lib/ad-targeting.ts
+        isSensitive: CAPI.config.isSensitive,
+        videoDuration: CAPI.config.videoDuration,
+        edition: CAPI.config.edition,
+        section: CAPI.config.section,
+        sharedAdTargeting: CAPI.config.sharedAdTargeting, // missing type definition
+        adUnit: CAPI.config.adUnit,
+    },
+    blocks: CAPI.blocks,
+    editionId: CAPI.editionId,
+    pillar: CAPI.pillar,
+    contentType: CAPI.contentType,
+    sectionName: CAPI.sectionName,
+    shouldHideReaderRevenue: CAPI.shouldHideReaderRevenue,
+    pageType: {
+        isMinuteArticle: CAPI.pageType.isMinuteArticle,
+        isPaidContent: CAPI.pageType.isPaidContent,
+    },
```

Comment:
I've always found `pageType` to be a needless abstraction (from DCR's point of view) so maybe later when we review this structure we can lose this?

Reply:
agreeded